qa/acceptance-tests/src/test/java/io/camunda/it/auditlog/AbstractAuditLogClientIT.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.auditlog;
+
+import static io.camunda.it.auditlog.AuditLogUtils.assignUserToTenant;
+import static io.camunda.it.auditlog.AuditLogUtils.createTenant;
+import static io.camunda.it.auditlog.AuditLogUtils.deployResource;
+import static io.camunda.it.auditlog.AuditLogUtils.startProcessInstance;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.client.CamundaClient;
+import io.camunda.client.api.command.ProblemException;
+import io.camunda.client.api.response.ProcessInstanceEvent;
+import io.camunda.client.api.search.enums.AuditLogCategoryEnum;
+import io.camunda.client.api.search.enums.AuditLogEntityTypeEnum;
+import io.camunda.client.api.search.enums.AuditLogOperationTypeEnum;
+import io.camunda.client.api.search.enums.AuditLogResultEnum;
+import io.camunda.qa.util.auth.Authenticated;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.model.bpmn.BpmnModelInstance;
+import io.camunda.zeebe.protocol.record.intent.ProcessInstanceCreationIntent;
+import io.camunda.zeebe.test.util.record.RecordingExporter;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+
+abstract class AbstractAuditLogClientIT {
```

Comment:
💭 Maybe a better name could be `BasicAuditLogClientIT` or `Base...` - it feels a bit strange that an `abstract` test has `@Test` case 🤔